src/view/.eslintrc

```diff
@@ -17,7 +17,11 @@
         "depth": 1
       }
     ],
-    "space-before-function-paren": [2, "never"],
+    "space-before-function-paren": [2, {
```

Comment:
had to fix this because eslint and prettier were fighting over spacing with `asyncArrow` functions

Reply:
You might want to use https://github.com/prettier/eslint-config-prettier. It looks like it's installed as a dependency already, but I _think_ `extends` is supposed to have a value of `prettier` in its array.